app/src/main/kotlin/com/flxrs/dankchat/chat/ChatAdapter.kt

```diff
@@ -706,6 +712,21 @@ class ChatAdapter(
     private fun String.toRequest(context: Context): ImageRequest = ImageRequest.Builder(context)
         .data(this)
         .build()
+
+    /** set background color, and enable/disable ripple (whether enable or disable should match the "clickability" of that message */
+    private fun TextView.setRippleBackground(@ColorInt backgroundColor: Int, enableRipple: Boolean = false) {
+        // The weird cast down here is because I can't write like
+        // if (background is RippleDrawable) { ... } else { ... }
+        // the compiler complains background is mutable and may have changed by the time it's accessed
+        val rippleBg = background as? RippleDrawable
+        if (rippleBg != null){ // background is expected set to RippleDrawable via XML layout
```

Comment:
```suggestion
        if (rippleBg != null) { // background is expected set to RippleDrawable via XML layout
```
Can you perhaps reformat the changed files?